tests/integration/delete/delete_integ_base.py

```diff
@@ -22,10 +23,19 @@ def base_command(self):
         return command
 
     def get_delete_command_list(
```

Comment:
This PR is identical to #3915 except here I've reverted the changes to the get delete command method to what it was initially to resolve integration tests failing in certain conditions on Windows machines.